frontend/src/component/Search/AdvancedSearchContentList.styl

```diff
@@ -0,0 +1,124 @@
+@import "~tracim_frontend_lib/src/css/Variable.styl"
+
+.advancedSearchContent
+  height 65px
+  width 100%
+  display flex
+  align-items center
+  &__typeAndTitle
+    display none
+  &__type
+    &__content
+      font-size fontSizeTitle
+      display flex
+      flex-direction column
+      align-items center
+      width 90px
+      place-content center
+      & > span
+        font-size fontSizeSmall
+    &__header
+      margin 15px 25px
+      text-align center
+      align-self center
+  &__title
+    padding 0px 5px
+    white-space nowrap
+    overflow hidden
+    text-overflow ellipsis
+    color fontColor
+    align-self center
+    flex 1
+    width 0
+  &__modification
+    align-self center
+    margin-inline-end 90px
+  &__information
+    color fontColor
+    align-self center
+    margin-inline-end 5px
+    width 160px
+    & > *
+      font-size fontSizeSmall
+    & > i
+      margin-inline-end 5px
+
+.advancedSearch__openMenu
+  & .timedEvent.advancedSearchContent__modification .timedEvent__operation
+    display none
+  & .advancedSearch__content .content .breadcrumbs
+    display none
+  & .advancedSearchContent .advancedSearchContent__title
+    padding-bottom 0
+
+// INFO - G.B. - 20210210 - The rules below (and its media queries) are in absolute position because they come from code that would have nested links.
+// To avoid this, the components have been placed outside the top link and displayed in the right place using absolute.
```

Comment:
what are those 2 links ? Is it the `<Link />` from ProfileNavigation and the one inside ListItemWrapper ?

Reply:
There are `<Link />` at AdvancedSearchContentList (to the content), Breadcrumbs (to each item) and ProfileNavigation (to users profile).

Reply:
precision: inside the children of ListItemWrapper

Reply:
yes, sry, the one inside ListItemWrapper and Breadcrumbs (to each item) / ProfileNavigation (to users profile).